src/mce_plugin.c

```diff
@@ -97,9 +122,182 @@ mce_plugin_display_state_handler(
     MceDisplay* display,
     gpointer plugin)
 {
-    mce_plugin_update_power(THIS(plugin));
+    mce_plugin_update_state(THIS(plugin));
+}
+
+static
+void
+mce_plugin_tklock_state_handler(
+    MceTklock* tklock,
+    void* plugin)
+{
+    mce_plugin_update_state(THIS(plugin));
+}
+
+static
+void
+mce_plugin_update_state(
+    McePlugin* self)
+{
+    const gboolean track_display_state = !self->always_on;
+    const gboolean track_lock_state = !self->always_on && self->require_unlock;
```

Comment:
Pondering about: if / where / how clearly it is stated that RequireUnlock=true is ignored when AlwaysOn=true?

Reply:
Yeap, I agree with @spiiroin , they should be tracked separately

Reply:
`AlwaysOn` is mostly (if not only) for debugging and was never meant to be exposed to the user (unlike `RequireUnlock` which can be imagined to be user-configurable). And yes, they contradict each other.